plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.ts

```diff
@@ -157,11 +162,15 @@ export class GoogleAnalytics implements AnalyticsApi {
    * pageview and the rest as custom events. All custom dimensions/metrics are
    * applied as they should be (set on pageview, merged object on events).
    */
-  captureEvent(event: AnalyticsEvent) {
+  captureEvent(event: AnalyticsEvent | NewAnalyticsEvent) {
     const { context, action, subject, value, attributes } = event;
     const customMetadata = this.getCustomDimensionMetrics(context, attributes);
 
-    if (action === 'navigate' && context.extension === 'App') {
+    const extensionId = context.extensionId || context.extension;
+    const category = extensionId ? String(extensionId) : 'app';
+
+    // The legacy default extension was 'App' and the new one is 'app'
+    if (action === 'navigate' && category.toLocaleLowerCase() === 'app') {
```

Comment:
Realized this might break in the new system since the extension ID might become `app/root` instead. Perhaps worth being careful not to introduce an extension context at that level, or we could tweak this check to check for an `app/` prefix too, or use the plugin ID.

Reply:
Updated to`category.toLocaleLowerCase('en-US').startsWith('app')`. Is it fine?